NuSpecs/Humanizer.Core.en-US.nuspec

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.12">
+    <id>Humanizer.Core.en-US</id>
+    <version>$version$</version>
+    <title>Humanizer Locale (en-US)</title>
+    <authors>Mehdi Khalili, Claire Novotny</authors>
+    <projectUrl>https://github.com/Humanizr/Humanizer</projectUrl>
+    <icon>logo.png</icon>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Humanizer Locale English (United States) (en-US)</description>
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <license type="expression">MIT</license>
+    <repository type="$RepositoryType$" url="$RepositoryUrl$" commit="$RepositoryCommit$" />
+    <language>en-US</language>
+    <dependencies>
+       <dependency id="Humanizer.Core" version="[$version$]" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="Humanizer\bin\Release\netstandard1.0\en-US\*.*" target="lib\netstandard1.0\en-US" />
+    <file src="Humanizer\bin\Release\netstandard2.0\en-US\*.*" target="lib\netstandard2.0\en-US" />
```

Comment:
needs a net6.0 target now -- please rebase on main

Reply:
I will gladly do it, if you confirm I've chosen the right approach right off the bat. See [here](https://github.com/Humanizr/Humanizer/pull/1068#discussion_r641011389). Otherwise I can push the changes I mentioned there.

Reply:
Ok, just rebased on main.